message_ix_models/data/transport/R11/ldv-inv_cost.csv

```diff
@@ -0,0 +1,851 @@
+# Fixed operation & maintenance cost of LDVs
+#
+# Source: Extracted from ldv-cost-efficiency-full.xlsx sheets named, e.g.
+"# ""MESSAGE_LDV_nam"" in cells C62:Q74. Cells that duplicated the preceding period"

```

Comment:
@r-aneeque FYI this file is malformed; there is a leading quotation mark `"` ahead of the comment character `#` at the start of this line. The code will not be able to read the file.